### PR TITLE
Convert to theforeman/actions

### PR DESCRIPTION
**Pull request by ekohl**
Currently a draft since it doesn't call the core extensions yet.

### CODE REVIEW

**Comment by ekohl** [comment]
I noticed there is a RuboCop config, but nothing calls it today. Or at least, when I activated it, it didn't pass.

File: .github/workflows/ruby_tests.yml [hunk 1]
```diff
@@ -1,79 +1,17 @@
...
-      - name: Run migrations
-        run: |
-          bundle exec rake db:create
-          bundle exec rake db:migrate
-      - name: Run plugin tests
-        run: |
-          bundle exec rake test:foreman_openscap
-      - name: Run selected core tests
-        run: bundle exec rake test:foreman_openscap_extensions
+  # TODO: bundle exec rake test:foreman_openscap_extensions
```
Comment (evgeni) [inline .github/workflows/ruby_tests.yml:12]: Mhh, `matrix_include` of the action doesn't allow us to extend the `tests` list? Sad.

Comment (ofedoren) [reply .github/workflows/ruby_tests.yml:12]: Could we do something like this https://github.com/theforeman/foreman_openscap/pull/557/files#diff-c105fe0c2711318227c0cf6c915452e4be538d8a7d39a7fa7f387fef15a63311R89? I mean that task is only for these files. Maybe for now we could include them in the main test task?

Comment (ekohl) [reply .github/workflows/ruby_tests.yml]: Done so now. I think your had an issue with globbing, not really sure.

**Review by ofedoren — APPROVED** [review]
Thanks, @ekohl, @evgeni!

**Pull request merged**